docs/src/rsh/step/index.md

```diff
@@ -396,7 +396,12 @@ wait(TIME);
 A @{defn("wait statement")}, written `{!rsh} wait(TIME);`, delays the computation until the `{!rsh} TIME` time argument passes.
 `{!rsh} TIME` must be pure and may only reference values known by the consensus state.
 
-In this example, `{!rsh} wait` is used so Bob doesn't have an advantage over Alice by making two separate `{!rsh} publish` statements at the same time. `{!rsh} wait` ensures that Alice has enough time to verify that Bob accepted the `{!rsh} wager` before beginning a [`{!rsh} race`](##guide-race).
+In this example, `{!rsh} wait` is used so Bob doesn't have an advantage over Alice by making two separate `{!rsh} publish` statements at the same time. 
+`{!rsh} wait` ensures that Alice has enough time to verify that Bob accepted the `{!rsh} wager` before beginning a [`{!rsh} race`](##guide-race). 
+
+:::note
+Learn more about the `{!rsh} race` expression in our guide, "@{seclink("guide-race")}."
```

Comment:
Do the "s look nice? Should the "." be after?

Reply:
The period before the close quote is correct. 
https://www.grammarly.com/blog/underline-or-italicize-book-titles/

I'm willing to remove the quotations, but since this is the name of the guide, the quotes are grammatically correct. Either way the comma before the opening quote should be removed.